config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
```

Comment:
- [OPTIONAL] Please use `postgresql` adapter moving forward. The reason is that it is a robust database, efficient and has other features that you will use frequently in future projects.